jobs/delorean/2.0/ews/Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+
+pipeline {
+    agent none
+
+    parameters {
+        // Example message:
+        // https://datagrepper.engineering.redhat.com/id?id=ID:messaging-devops-broker01.web.prod.ext.phx2.redhat.com-43656-1547130290231-6:586552:0:0:1&is_raw=true&size=extra-large
+        string defaultValue: '', description: '', name: 'CI_MESSAGE', trim: true
+    }
+
+    options {
+        disableConcurrentBuilds()
+        buildDiscarder logRotator(numToKeepStr: '3', daysToKeepStr: '8')
+    }
+
+    stages {
+        stage('Send mail') {
+            steps {
+                script {
+                    final Map msg = readJSON text: CI_MESSAGE
+                    // Sending this mail is just to test things for now
+                    // TODO Replace with pr creation
+                    mail(
+                        subject: "New build of ${msg.info.package_name} created in Brew",
+                        to: 'lfitzger@redhat.com',
+                        body: 
```

Comment:
This needs an opening square bracket to match the closing one on L29. The body parameter here expects a string, so that's what's happening on L29 -- the list of lines is joined into a string. This is a little prettier than constructing the string upfront with `\n` characters and stuff.